### DISCUSSION TITLE
Signal to track the exported variable

### DISCUSSION THREAD

**Discussion by JekSun97** [godotengine/godot-proposals#8718] [Engine Core]
I'm working on a project where there are characters, and using an exported variable of the enum type, I want to change the armor/clothing on the characters from the editor, add a signal to change the exported variables.
`func _change_var_export ( VarName: String, NewValue: Variant)`

Please also add this for version 3.6

**Comment by AThousandShips** [comment]
Duplicate of https://github.com/godotengine/godot-proposals/issues/4868
